src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void registerValidUsers_Ok() {
+        registrationService.register(new User("validLogin", "validPass", 18));
+        registrationService.register(new User("ValidLogin@gmail.com", "Valid_pass98", 18));
+        registrationService.register(new User("validlogin_98", "valid_Pass3", 65));
```

Comment:
you basically only check that no exception is thrown, it's not enough, let's visit the checklist:
![image](https://github.com/mate-academy/jv-registration-validation/assets/49767049/cf05eef6-7a70-4288-9a38-dc358a0c8a55)
sorry this was not pointed out earlier